### DISCUSSION TITLE
[Feature Request] Have option to enable Wide Angle Lens

### DISCUSSION THREAD

**Discussion by NMBlanco** [rodit/SnapMod#183] [Ideas & Feature Suggestions]
It would be cool if you could implement the wide angle lens in the camera ui of snapchat.
Like an option to enable it here:
![Screenshot_20220603-162323](https://user-images.githubusercontent.com/12681440/171873525-b29f0207-7788-4b39-ac09-569e6b663778.png)

**Comment by RevealedSoulEven** [comment]
then add macro too... but how will it check whether the phone have macro or wide angle lens or not

**Comment by rodit** [comment]
I don't have any wide angle lens devices to test on so this is virtually impossible for me to develop unfortunately.
